### PR TITLE
Sponsors page second iteration

### PR DESCRIPTION
**Pull request by skrings**
FYI @janz93 

### Tasks
- [x] Add links to careers 
- [x] Add text for phraseapp 
- [x] Add text for wer liefert was
- [ ] Update text for sum.cumo 
- [x] Update text for x-ion
- [x] Update text for wer liefert was

### CODE REVIEW

File: sponsors.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ Please <a href="mailto:sponsoring@rubyunconf.eu">contact us</a> about sponsoring
...
     No need for a CV - simply import your LinkedIn profile, have a quick chat with your personal Talent Advocate to let them know what you are looking for and when you're ready, go live for a 4-week session on the platform while you are open to finding a new job.<br/>
     On talent.io you state your desired work location, tech stack, salary expectation and desired new position upfront, so that you are contacted only with offers that are relevant to your prior experience, and where you want to go next.<br/>
-    There are no middle-men, receive offers from hiring companies directly and transparently. <a href="https://careers.talent.io/" target="_blank">Join us</a>!
+    There are no middle-men, receive offers from hiring companies directly and transparently. <br/>
+    If you're interested in being contacted directly by top tech companies for offers, find <a href="https://www.talent.io/users/sign_up?utm_source=event&utm_campaign=RubyUnConf&utm_content=talentioBlurb" target="_blank">talent.io</a> signup here!
+    Join us</a>!
```
Comment (skrings) [inline sponsors.md]: @janz93 I guess you wanted to replace `Join us` with `signup here` -  now we have both 😄

Comment (janz93) [reply sponsors.md]: @skrings Yes it was a copy and paste error 🙈

Comment (janz93) [reply sponsors.md]: @janz93 add proper link

**Comment by skrings** [comment]
@janz93 Will update the text for sum.cumo in a follow up pull request.

**Pull request merged**